Sources/Parsing/Documentation.docc/Articles/Parsers/Bool.md

```diff
@@ -31,16 +31,17 @@ UTF-8 code units (see <doc:StringAbstractions> for more info).
 
 Typically Swift can choose the correct overload by using type inference based on what other parsers
 you are combining `Bool.parser()` with. For example, if you use `Bool.parser()` with a
-`Substring` parser, say the literal "," parser (see <doc:String> for more information), Swift
+`Substring` parser, say the literal `","` parser (see <doc:String> for more information), Swift
 will choose the overload that works on substrings:
 
 ```swift
-try Parse {
+let parser = Parse {
   Bool.parser()
   ","
   Bool.parser()
 }
-.parse("true,false") // (true, false)
+
+try parser.parse("true,false") // (true, false)
```

Comment:
Thanks! Stephen brought this to my attention before merging #171 but looks like I didn't update all the spots.